src/header/Header.js

```diff
@@ -126,27 +91,23 @@ Header.defaultProps = {
 };
 
 const styles = StyleSheet.create({
-  innerContainer: {
-    flex: 1,
+  container: {
+    backgroundColor: colors.primary,
+    borderBottomColor: '#f2f2f2',
+    borderBottomWidth: StyleSheet.hairlineWidth,
+    paddingHorizontal: 10,
+    paddingTop: getStatusBarHeight(),
     flexDirection: 'row',
     justifyContent: 'space-between',
     alignItems: 'center',
+    height: Platform.OS === 'ios' ? 44 : 56 + getStatusBarHeight(),
```

Comment:
Why did the height change from the original?

Reply:
I just took the height from material and iOS design. And before, the height was calculated differently :  
```js
height: Platform.OS === 'ios' ? 70 : 70 - getStatusBarHeight(),	
  ...Platform.select({	
  ios: {	
    paddingTop: getStatusBarHeight(),	
  },	
}),
```

Reply:
iOS doesn't account for the height of `<StatusBar />` in it's calculation, but android does. So we had to explicitly remove the status bar height on android

Reply:
I don't know... It works well on both iOS and Android with this ...

Reply:
Cool alright then. Would be good if you can do before and after pictures and drop em in this PR so we can look back and see when it was changed.

Reply:
And it's `(Platform.OS === 'ios' ? 44 : 56`) + getStatusBarHeight() with () 🙁, sorry for that

Reply:
Just updated the pr comment with screenshots !

Reply:
Sweet! Snapshots bro :D